### ISSUE TITLE
"Why Elm?" section: condense "lots of rules" with "some bugs are impossible"

### ISSUE THREAD

**Issue by avh4** [elmbridge/curriculum#35]
The few paragraphs starting at

> And the language has lots of rules! ...

I think this part could be shorter and made to sound more motivating if the list of rules and the list of benefits were combined into one list (since each rule corresponds to preventing a certain type of bug).

**Comment by raorao** [comment]
My initial separation was intentional, as I feel like intro to elm materials can be somewhat cheerleader-y about the language's strictness. I wanted to faithfully describe the rules before explaining why they are good, so that readers can have their own reaction to them. 

That said, I could definitely be missing a rewrite here. Would love to see an alternate draft.

**Comment by siruguri** [comment]
I would like to work on an alternate draft, so I'll piggy back on this issue but let me first describe my motivations, so you can let me know if this merits a separate issue (or if the ideas themselves have already been though through.)

My idea has been to separate the "Elm brings benefits of statically compiled language programming to front end" parts, from the "Elm gives you a framework like React/JSX" part. I think these are two distinct points to using Elm, and it would help to clarify that.

Additionally, I want to take this opportunity to make the language a bit more "active,"  in the sense largely conveyed in [this guide](http://www.austincc.edu/history/inres10a4style.html) or perhaps [this one about writing persuasive essays](https://owl.english.purdue.edu/owl/resource/685/05/).

I'll ping folks if I don't hear back in a day or two, before I proceed to work on a PR.

**Comment by raorao** [comment]
> My idea has been to separate the "Elm brings benefits of statically compiled language programming to front end" parts, from the "Elm gives you a framework like React/JSX" part

This is the intention of the current draft, as the "framework" stuff is mostly explained in "Building User Interfaces." If you feel like "Why Elm?" talks too much about frameworks, I would definitely appreciate a PR to fix. 

> Additionally, I want to take this opportunity to make the language a bit more "active," in the sense largely conveyed in this guide or perhaps this one about writing persuasive essays.

As for the language not being "active", I would need to see some examples of fixes to be convinced. The voice of curriculum is intentionally passive -- this is not a piece of argumentative writing, and as such should not emphasize the subject over statements of fact. My sense is that curriculum should be expository, not argumentative. (though this may simply may be my bias, as I was trained in news writing, which roughly fits in to the expository category of the classical rhetorical model). 

I would not be surprised if I was guilty of poorly deploying passive constructions (e.g. "mistakes were made" vs "we made a mistake") and perhaps that is what you mean by making the text more "active." Again, I would love to see some specific examples of possible issues.

**Comment by siruguri** [comment]
Fair enough - I'll submit my suggestions as a PR and you can feel free to
keep/merge/reject as appropriate.

Certainly the expository style came through, but the "we'd argue that ...
can be fruitful" phrasing towards the end had me wondering. So I'll take a
shot, and tell me what you think.
